### ISSUE TITLE
Clash between play and download!

### ISSUE THREAD

**Issue by rohanricky** [the-ethan-hunt/B.E.N.J.I.#83]
Since youtube play feature is above download it is causing few bugs errors.
For example, if I type download Coldplay Paradise, it contains play so it is playing the video. Similarily, Chainsmokers Bloodstream contains stream.

**Comment by the-ethan-hunt** [comment]
Nice spotting the bug, @rohanricky !

**Comment by the-ethan-hunt** [comment]
This issue is open for discussion: @aribalam , @kb-studios @Yatish04 . You all are open to discuss this issue

**Comment by kb-studios** [comment]
As this can be a very broad issue, can we start off using NLP gradually, maybe usinh nltk? We can do basic tokenizing.

**Comment by the-ethan-hunt** [comment]
Good thinking @kb-studios ! But tokenizing  requires a knowledge of NLP. But though won't fix for sometime, this is a good issue for discussion

**Comment by Yatish04** [comment]
Yeah we have to use nlp at the end of the day.😀 So as of now it's limited

**Comment by rohanricky** [comment]
For now we can reverse it's order to get it working :') or since we are using download and play at starting only we can use .startwith() for this also. This is just a quick fix eventually we can work on nlp.

**Comment by the-ethan-hunt** [comment]
You are right @rohanricky in this sense. We can use that temporarily. You can work on this issue. but we are not closing it
